### ISSUE TITLE
[gatsby-plugin-manifest] Automatic mode error: Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by alampros** [gatsbyjs/gatsby#4865]
## Description

When running the plugin with no `icons` property in the options (only `icon` for the source image), `gatsby build` throws a build error.

```
Error: Cannot read property '0' of undefined

  - render-page.js:22888 Object.exports.onRenderBody
    render-page.js:22888:24
```

### Steps to reproduce

1. Configure `gatsby-plugin-manifest` as described in [Automatic Mode](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-manifest#automatic-mode)
2. Run `gatsby build`

### Expected result

The plugin should use the `defaultIcons` from:

https://github.com/gatsbyjs/gatsby/blob/362f78a3e16830770b68023e8e939992e47a3cb5/packages/gatsby-plugin-manifest/src/gatsby-node.js#L6-L47

...and
1. generate the default set of images
2. populate the `icons` property of the `manifest.json` with the default icons

### Actual result

Build throws error:

```
Error: Cannot read property '0' of undefined

  - render-page.js:22888 Object.exports.onRenderBody
    render-page.js:22888:24
```

### Environment

* Gatsby version (`npm list gatsby`): 1.9.246
* gatsby-cli version (`gatsby --version`): 1.9.246
* Node.js version: v8.11.0
* Operating System: macOS 10.13.4

### File contents (if changed):

`gatsby-config.js`: 

```js
    {
      resolve: 'gatsby-plugin-manifest',
      options: {
        name: pkg.title,
        short_name: pkg.shortName,
        start_url: '/',
        background_color: '#DFE7F7',
        theme_color: '#00A3DB',
        display: 'minimal-ui',
        icon: 'src/icon.png',
      },
    },
    'gatsby-plugin-offline',
```

**Comment by caki0915** [comment]
+1 on this one. I get the same error.

**Comment by pieh** [comment]
@caki0915  There is PR with fix for this #4867 , but in meantime you can downgrade to `gatsby-plugin-manifest@1.0.15`

**Comment by KyleAMathews** [comment]
Fix is published to NPM so update to the latest gatsby-plugin-manifest.
